01-start/03-begin.html.md

```diff
@@ -565,7 +560,7 @@ As you now have all the tools and knowledge required to be able to create the bl
 
 Congratulations! You now possess all the foundations required to be able to write amazing and powerful web applications like those already in our [Showcase](/docpad/showcase). To recap, you now know how to:
 
-- write documents in any language, markup, pre-processor, templating engine, whatever you wish, by installing the necessary plugin for it and changing the extensions of the document
+- write `render` files in any language, markup, pre-processor, templating engine, whatever you wish, by installing the necessary plugin for it and changing the extensions of the document
```

Comment:
That was exactly what happened. With two separate directories it's much harder to detect that you have two files that generate to the same output file.

Unless there's good reason to allow having such an error-prone configuration, it should be disallowed.